### PR TITLE
Make manual valves work

### PR DESCRIPTION
**Pull request by wrexbe**
About PR
- Change manual valve icon to correct one
- Manual valve changes appearance when it is opened, or closed.
- Manual valve now blocks, or allows air to pass based on if it is opened or closed.

**Changelog**
:cl:
- add: Manual valves change appearance when you open, or close them.
- fix: Manual valves now correctly block, or allow air.

### CODE REVIEW

**Review by ElectroJr — COMMENTED** [review]
I don't really know enough about piping or node networks to comment on the code, but it seems to work as advertised.

File: Content.Server/Atmos/Piping/Binary/EntitySystems/GasValveSystem.cs [hunk 1]
```diff
@@ -52,18 +52,30 @@ private void OnActivate(EntityUid uid, GasValveComponent component, ActivateInWo
...
                 Toggle(uid, component);
-                SoundSystem.Play(Filter.Pvs(component.Owner), component._valveSound.GetSound(), component.Owner, AudioHelpers.WithVariation(0.25f));
+                SoundSystem.Play(Filter.Pvs(component.Owner), component.ValveSound.GetSound(), component.Owner, AudioHelpers.WithVariation(0.25f));
             }
         }
 
         public void Set(EntityUid uid, GasValveComponent component, bool value)
         {
             component.Open = value;
+            var appearance = EntityManager.GetComponentOrNull<AppearanceComponent>(component.Owner);
```
Comment (ElectroJr) [inline Content.Server/Atmos/Piping/Binary/EntitySystems/GasValveSystem.cs:62]: Why not just TryComp & set appearance data?
Or is there some reason it should only be called if this also has a `NodeContainerComponent`?

**Comment by metalgearsloth** [comment]
@Zumorica 

Bikeshedding but having the light as another layer that's unshaded would be neat in future.

**Review by Zumorica — APPROVED** [review]
I guess this is a way to do it.

**Review by ShadowCommander — APPROVED** [review]

**Pull request merged**